src/main.h

```diff
@@ -507,4 +507,10 @@ extern CCoinsViewCache *pcoinsTip;
 /** Global variable that points to the active block tree (protected by cs_main) */
 extern CBlockTreeDB *pblocktree;
 
+/** 
+ * While checking, GetBestBlock() refers to the parent block. (protected by cs_main)
```

Comment:
Yeah, I just moved the comments from the cpp instead of moving the implementation of the new function. I've just added a line to the doc. Should I leave the rest here or move it back to the implementation?